starter-project-web-vue2/package.json

```diff
@@ -20,9 +20,11 @@
     "@nuxtjs/auth-next": "5.0.0-1624817847.21691f1",
     "@nuxtjs/axios": "^5.13.6",
     "core-js": "^3.19.3",
+    "moment": "^2.29.4",
     "nuxt": "^2.15.8",
     "vue-server-renderer": "^2.6.14",
     "vue-template-compiler": "^2.6.14",
+    "vuelidate": "^0.7.7",
```

Comment:
Vuetify form can validate user input. An additional validation package won't be necessary.